Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -534,8 +546,44 @@ namespace O3DE::ProjectManager
         }
     }
 
+    void GemCatalogHeaderWidget::GemCartShown(bool state)
+    {
+        m_showGemCart = state;
+        repaint();
+    }
+
     void GemCatalogHeaderWidget::ReinitForProject()
     {
         m_filterLineEdit->setText({});
     }
+
+    void GemCatalogHeaderWidget::paintEvent([[maybe_unused]] QPaintEvent* event)
```

Comment:
another potential option here might have been to use a positioned background svg image - probably not worth the time right now to try.

Reply:
Personally I think this is better, easily adjustable and dynamically programmatically defined